openshift-ci/step-registry/capz-test-generate-yamls-commands.sh

```
@@ -4,8 +4,56 @@ set -o nounset
+echo "================================"
+
 # Phase 04: Generate YAMLs
 # Generates credential and cluster YAML manifests for deployment.
 # Produces JUnit XML in ${ARTIFACT_DIR} for Prow to collect.
 export TEST_RESULTS_DIR="${ARTIFACT_DIR}"
-make _generate-yamls RESULTS_DIR="${ARTIFACT_DIR}"
+script -e -q -c "make _generate-yamls RESULTS_DIR=\"${ARTIFACT_DIR}\"" /dev/null
+
+# Diagnostic: show ARO_REPO_DIR contents after generation
+echo "=== ARO_REPO_DIR contents ==="
+ls -la "${ARO_REPO_DIR}/" | head -20
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)